### ISSUE TITLE
Perl auto-completion and code intel does not work when loading the FindBin module and importing $Bin and/or $Script. 

### ISSUE THREAD

**Issue by casimirloeber** [Komodo/KomodoEdit#3472]
### Short Summary

Perl auto-completion and code intel does not work when loading the FindBin module and importing $Bin and/or $Script. 

**Works:**

`use FindBin;`

**Doesn't work:**

`use FindBin qw($Bin $Script);`
`use FindBin qw($Script);`

Take a look at the attached file and comment out the different options for loading FindBin and then go a few lines below it and type 'prin', it should give you code intel on the print function. This does not work when importing $Bin and/or $Script variables

[test.txt](https://github.com/Komodo/KomodoEdit/files/1721025/test.txt)
![feb-13-2018 20-02-59](https://user-images.githubusercontent.com/3631995/36165662-fdc310d6-10f8-11e8-9e50-795623ebae88.gif)

### Platform Information

Komodo IDE, version 11.0.2, build 90813, platform macosx. Built on Mon Dec 4 09:43:06 2017.
MacOS High Sierra 10.13.3
perl 5.22.0

**Comment by mitchell-as** [comment]
Thanks for the report!

**Comment by casimirloeber** [comment]
Any update on this one by any chance?

**Comment by mitchell-as** [comment]
No, sorry. This ticket is not targeted for a particular release yet.

**Comment by casimirloeber** [comment]
Is there anything I can do to help bump it up? Is this something I could potentially work on? Can I pay $ to have it included ;)

Regardless, thanks for an awesome product!!!

**Comment by mitchell-as** [comment]
I appreciate your enthusiasm! Unfortunately, at this time only we can work on this issue. We'll let you know if this changes though :)
